src/main/drivers/rtc.c

```diff
@@ -0,0 +1,136 @@
+#include "common/printf.h"
```

Comment:
Most of the functions in `rtc.c` seem to do with formatting time and time conversion. Such functions don't belong in a device driver, they should be in `common/time.c`

Reply:
I second this. Also definitions of `timestamp_t`, `date_time_t` and RTC handling functions should probably go to `common/time.h`

Reply:
@martinbudden A left over from the initial version, which was using the hardware RTC. I'll move everything somewhere where it makes more sense.